website/docs/migrating-from-docker/managing-docker-compatibility.md

```diff
@@ -0,0 +1,46 @@
+---
+sidebar_position: 5
+title: Managing docker compatibility
+description: Covers configurations available to manage docker compatibility
+keywords: [podman desktop, podman, docker, compatibility]
+tags: [managing-docker-compatibility]
+---
+
+# Managing docker compatibility
+
+With Podman Desktop, you can configure a Docker-compatible environment to run your Docker applications on a Podman engine, rather than on a Docker engine.
+
+As a developer, you can:
+
+- Set the Docker compatibility mode to emulate a Docker API socket connection to the `var/run/docker.sock` default path.
+- Use all Docker commands with Podman. For example, you can use the `podman run` command in place of `docker run` command to start a container.
+- Use Docker Compose commands with Podman. For example, you can use the `podman compose up` command in place of `docker compose up` command to start your Docker application. You can place your Docker Compose file in a _/var/compose/_ directory so that Podman can access it.
```

Comment:
Correct me if I am wrong, but part of the power of the `Compose compatibility` is that you can run `docker compose up` and it will run on the Podman engine underneath